packages/filesystem/src/browser/file-dialog/file-dialog-service.ts

```diff
@@ -83,14 +83,11 @@ export class DefaultFileDialogService implements FileDialogService {
             resource: new URI(await this.environments.getHomeDirUri()),
             isDirectory: true
         };
-        if (folder) {
```

Comment:
I removed the `if (folder) { ... }` condition since folder is always defined.
I can revert if necessary.